### PR TITLE
230710 Company metrics endpoint changelog

### PR DESCRIPTION
**Pull request by pzaichkina**
# Description

Creation of a new changelog entry for the GET /metrics/companies deprecation.

## Type of change

Please delete options that are not relevant.

- [X] New document(s)/updating existing
- [ ] Fixes
- [ ] Styling
- [ ] Bug fix (non-breaking change which fixes an issue)

### CODE REVIEW

File: blog/230710-deprecation-companies-metrics.md [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+tags: ["Deprecation"]
+authors: pzaichkina
+---
+
+On July 10, 2023, Codat will deprecate the `GET /metrics/companies` endpoint previously used to display the status of companies on the Portal dashboard. 
+
+<!--truncate-->
+
+It provided the following status metrics:
+- Currently linked,
```
Comment (mcclowes) [inline blog/230710-deprecation-companies-metrics.md:13]:
```suggestion
- Currently linked
```

File: blog/230710-deprecation-companies-metrics.md [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+- No longer linked,
+- Not yet linked,
+- All time linked. 
+
+The internal logic of the dashboard has changed, and the endpoint is no longer in use. After the change is implemented, any requests made to `GET /metrics/companies` will return a 404 error. 
+
+## Action required​
+
+If you are using the current `GET /metrics/companies` endpoint to perform monitoring outside of the Portal dashboard, make sure to use alternative API calls instead. You can obtain data connection statuses from the `GET /companies`.
```
Comment (d-laing) [inline blog/230710-deprecation-companies-metrics.md:22]:
```suggestion
If you are using the current `GET /metrics/companies` endpoint to perform monitoring outside of the Portal dashboard, make sure to use alternative API calls instead. You can obtain data connection statuses from the `GET /companies` endpoint. 
```

**Review by d-laing — CHANGES_REQUESTED** [review]
Just one minor thing.

**Review by d-laing — APPROVED** [review]
Looks good.

**Pull request merged**